x/oracle/types/params.go

```diff
@@ -32,9 +32,12 @@ const (
 var (
 	DefaultVoteThreshold = sdk.NewDecWithPrec(50, 2) // 50%
 	DefaultRewardBand    = sdk.NewDecWithPrec(2, 2)  // 2% (-1, 1)
-	DefaultTobinTax      = sdk.NewDecWithPrec(25, 4) // 0.25%
 	DefaultWhitelist     = DenomList{
-		{Name: UmeeDenom},
+		{
```

Comment:
So a few things:

1. We shouldn't store ATOM in the default list
2. We should only store umee
3. ATOM won't be interpreted as "atom", it'll be the IBC denom, e.g. `ibc/CCFAF...`